lakefile.lean

```diff
@@ -56,6 +58,32 @@ lean_lib Cache where
 lean_exe cache where
   root := `Cache.Main
 
+partial def pipeLines (i o : IO.FS.Stream) : BaseIO Unit := do
+  if let .ok ln ← i.getLine.toBaseIO then
+    unless ln.isEmpty do
+      discard <| o.putStr ln |>.toBaseIO
+      pipeLines i o
+
+def noCacheGet := get_config? noCacheGet |>.isSome
```

Comment:
```suggestion
-- When run as `lake build -KnoCacheGet`, do not invoke `lake exe cache get` automatically.
def noCacheGet := get_config? noCacheGet |>.isSome
```
Just to provide a clue for people who want to opt out. Most people won't know that `-K` is a thing.